sos/report/plugins/microk8s.py

```diff
@@ -28,6 +29,22 @@ class Microk8s(Plugin, UbuntuPlugin):
     packages = ('microk8s',)
 
     microk8s_cmd = "microk8s"
+    kube_cmd = "microk8s kubectl"
+
+    files = (
+        "/var/snap/microk8s/current/credentials/client.config"
```

Comment:
in order for this tuple to work, you need to add a comma at the end

However, this tuple is about triggering the plugin, and not about collecting file config. I would think having the `microk8s` in the `packages` tuple should suffice.

If we need to collect the config file, then we need to add that to the `setup()` function and add

```python
self.add_copy_spec("/var/snap/microk8s/current/credentials/client.config")
```

also noting that we may need to add a `postproc()` function if we need to mask any private information like passwords and private kets from that config file

Reply:
Hi @arif-ali thaks! Missed the comma indeed :D

re:add_copy_spec
Do I need it? Just asking because we are not doing it in `sos/report/plugins/kubernetes.py::UbuntuKubernetes`

Reply:
The UbuntuKubernetes is there for standard kubernetes gathering.

With this plugin dedicated for `microk8s`, it may be useful to gather that file. Is it useful to gather that file, if so, then we can add it as a `add_copy_spec`, ensuring any obfuscation is done wrt any secrets

Reply:
Added the copy_spec